ompi/mca/mtl/ofi/mtl_ofi_component.c

```diff
@@ -626,6 +626,9 @@ ompi_mtl_ofi_component_init(bool enable_progress_threads,
         hints->domain_attr->threading = FI_THREAD_DOMAIN;
     }
 
+    if ((MTL_OFI_TAG_AUTO == ofi_tag_mode) || (MTL_OFI_TAG_FULL == ofi_tag_mode))
```

Comment:
Style nit: can you put `{}` around the block?  It's Open MPI style that we put `{}` around _all_ blocks, even 1-line blocks.

Thanks!

Reply:
Done, pushed update.